test/acts_as_authentic_test/session_maintenance_test.rb

```diff
@@ -92,5 +139,14 @@ def test_resetting_password_when_logged_out
       assert UserSession.find
       assert_equal ben, UserSession.find.record
     end
+
+    private
+
+    # This method makes sure that the config is always correct
+    # before each test
+    def setup_config(after_create = true, after_password_change = true)
+      User.log_in_after_create = after_create
+      User.log_in_after_password_change = after_password_change
+    end
```

Comment:
This is not what I meant by a `setup` method. Minitest has a method named `setup` (http://ruby-doc.org/stdlib-2.0.0/libdoc/minitest/rdoc/MiniTest/Unit/TestCase.html#method-i-setup) that automatically runs before every test. Do you think that would be useful here? It seems to me that most tests expect these options to be set to true, so that's what I'd put in `setup` and then the tests that expect false can set that in the test itself.